tests/integration/test_integration_gnmi.py

```diff
@@ -36,3 +39,40 @@ async def test_integration_get(service):
     )
     assert isinstance(response, gnmi.proto.GetResponse)
     validate_default_interfaces_get(response)
+
+
+async def test_integration_update_set_string(service):
```

Comment:
Is the target reseted between to tests?

Reply:
Since its a fixture with test scope, it is unique for each test.

Reply:
Yeah, I got it once I used it in the switch synchronizer. So we can easily add a tests for delete also.